src/app/search/SearchFilter.tsx

```diff
@@ -165,6 +165,9 @@ class SearchFilter extends React.Component<Props, State> {
         onClear={this.clearFilter}
       >
         <>
+          <span className="filter-match-count">
+            {t('Header.FilterMatchCount', { count: filteredItems.length })}
```

Comment:
It looks like you meant to translate `FilterMatchCount_plural` here (and should there be a check on whether to use that or `FilterMatchCount`?).

Reply:
@48klocs i18n will automatically add _plural to the key if count is greater than 1